### PR TITLE
Genericsql  bugs: mysql.user table is available and context fixed

### PR DESCRIPTION
**Pull request by ohaim1008**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/23354, https://github.com/demisto/etc/issues/23575

## Description
Fixing 5 bugs:
1. Context outputs not using skip and limit.
2. Tables_in_mysql (such as user) were not available due to unreadable values in the table.
3. In case there are connection parameters and we are using mssql db, the connection string should not contain "?" twice.
4. When query doesn't return results there we get the error "list index out of range". changing the error by including the stack trace and the query.
5. Changing docker image in order to support Oracle db.

### CODE REVIEW

**Comment by ohaim1008** [comment]
Bug fixed with same docker image.

File: Packs/GenericSQL/Integrations/GenericSQL/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+-
```
Comment (bakatzir) [inline Packs/GenericSQL/Integrations/GenericSQL/CHANGELOG.md:2]: please add a changelog message

**Comment by bakatzir** [comment]
@roysagi can you help us with the CHANGELOG?

**Comment by bakatzir** [comment]
@roysagi Orel added a few more lines to the CHANGELOG. please have a 2nd look.

**Comment by ohaim1008** [comment]
Build is failing due to docker image change which is needed in this PR.
Please consider force merge. @bakatzir

**Comment by bakatzir** [comment]
@ohaim1008 Please file an issue for a fix in the demisto-sdk validation.

**Pull request merged**